support-frontend/assets/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx

```diff
@@ -86,22 +84,20 @@ interface PropsFromParent {
 	paymentRequestObject: PaymentRequest | null;
 	setPaymentRequestObject: (paymentRequestObject: PaymentRequest) => void;
 	amount: number;
+	contributionType: ContributionType;
 	stripeAccount: StripeAccount;
 	stripeKey: string;
 	renderPaymentRequestButton: RenderPaymentRequestButton;
 }
 
 const mapStateToProps = (state: State, ownProps: PropsFromParent) => ({
-	selectedAmounts: state.page.form.selectedAmounts,
-	otherAmounts: state.page.form.formData.otherAmounts,
```

Comment:
before #3363 it was getting this prop both from it's parent & from redux! I think that means it just uses the redux one? either way, it doesn't need it so now it's gone!